kuma/wiki/models.py

```diff
@@ -1775,13 +1775,73 @@ def translation_age(self):
 
 
 class RevisionIP(models.Model):
-    """IP Address for a Revision."""
-    revision = models.ForeignKey(Revision)
-    ip = models.CharField(max_length=40, editable=False, db_index=True,
-                          blank=True, null=True)
-
+    """
+    IP Address for a Revision including User-Agent and
+    """
+    revision = models.ForeignKey(
+        Revision
+    )
+    ip = models.CharField(
+        _('IP address'),
+        max_length=40,
+        editable=False,
+        db_index=True,
+        blank=True,
+        null=True,
+    )
+    user_agent = models.TextField(
+        _('User-Agent'),
+        editable=False,
+        blank=True,
+    )
+    referrer = models.TextField(
+        _('HTTP Referrer'),
+        editable=False,
+        blank=True,
+    )
     objects = RevisionIPManager()
```

Comment:
Do the new fields user_agent and referrer need to be scrubbed in scripts/anonymize.sql?